pegasus/sites.v3/code.org/public/educate/csc/index.haml

```diff
@@ -50,74 +50,46 @@ theme: responsive_full_width
         %span •
         =hoc_s(:teacher_label_csc)
 
-%section#csc-modules
+%section.bg-neutral-light.centered
   .wrapper
-    %h2=hoc_s(:heading_find_a_module)
+    %h2=hoc_s(:csc_explore_curriculum)
+    %p=hoc_s(:csc_explore_desc)
+    %hr
+    %h3=hoc_s(:csc_find_the_right_fit)
+    .button-wrapper.centered
+      %a.link-button{href: "#csc-week-long-modules"}
+        =hoc_s(:csc_week_long_modules)
+      %a.link-button{href: "#csc-hour-long-lessons"}
+        =hoc_s(:csc_hour_long_lessons)
+
+%section#csc-week-long-modules
+  .wrapper
+    %h2=hoc_s(:csc_week_long_modules)
     %p=hoc_s(:csc_page_find_a_module_desc)
-    .action-block__wrapper.action-block__wrapper--three-col
+    = view :carousel_csc_week_long
+
+%section#csc-hour-long-lessons
+  .wrapper
+    %h2=hoc_s(:csc_hour_long_lessons)
+    %p=hoc_s(:csc_hour_long_lessons_desc)
+    .action-block__wrapper.action-block__wrapper--two-col
       .action-block.action-block--one-col.flex-space-between
         .content-wrapper
           %p.overline=hoc_s(:module_subject_english_language_arts)
-          %h3=hoc_s(:module_name_coding_w_poetry)
-          %img{src: "/images/csc-module-coding-w-poetry.png", alt: ""}
+          %h3=hoc_s(:tutorial_poemart_name)
+          %img{src: "/images/csc-module-poem-art.png", alt: "", style: "height: 100%, max-width: 430px"}
```

Comment:
These inline styles can be removed: `, style: "height: 100%, max-width: 430px"`
```suggestion
          %img{src: "/images/csc-module-poem-art.png", alt: ""}
```

Reply:
Ahh good catch. I eventually just cropped the image but forgot to remove this! Marking resolved now